src/NHibernate/Persister/Entity/AbstractEntityPersister.cs

```diff
@@ -4491,5 +4520,49 @@ public string GetInfoString()
 			return MessageHelper.InfoString(this);
 		}
 		#endregion
+
+		internal SqlString GenerateSequentialSelect(ILoadable persister)
```

Comment:
The argument should be `AbstractEntityPersister`.

Reply:
Done.